src/oc_erchef/apps/oc_chef_wm/src/oc_chef_action_queue_config.erl

```diff
@@ -0,0 +1,92 @@
+%% Copyright 2012-2014 Chef Software, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+
+-module(oc_chef_action_queue_config).
+
+-export([
+         get_rabbit_management_pool_name/0,
+         get_rabbit_management_pool_setting/0,
+         get_rabbit_queue_monitor_setting/2,
+         set_rabbit_management_setting/2,
+         set_rabbit_queue_monitor_setting/2,
+         set_app_value/5
+        ]).
+
+-define(SERVER, ?MODULE).
+-define(POOLNAME, rabbitmq_management_service).
```

Comment:
[minor] Perhaps `rabbitmq_actions_management_service` to match `rabbitmq_index_management_service`